src/tags/object/Video/Video.js

```diff
@@ -107,129 +103,80 @@ const Model = types
-          } else {
-            self.handleSyncPlay();
-          }
-        }
-        // Audio v1,v2
-        else {
-          Super.triggerSyncPlay();
-        }
-      },
+    ////// Incoming
 
-      triggerSyncPause() {
-        // Audio v3
-        if (isFFDev2715) {
-          if (self.syncedObject) {
-            Super.triggerSyncPause();
-          } else {
-            self.handleSyncPause();
-          }
-        }
-        // Audio v1,v2
-        else {
-          Super.triggerSyncPause();
-        }
-      },
+    registerSyncHandlers() {
+      ['play', 'pause', 'seek'].forEach(event => {
+        self.syncHandlers.set(event, self.handleSync);
+      });
+      self.syncHandlers.set('speed', self.handleSyncSpeed);
+    },
 
-      handleSyncSeek(time) {
-        // Audio v3
-        if (isFFDev2715) {
-          if (self.syncedDuration && time >= self.syncedDuration) {
-            self.ref.current.currentTime = self.ref.current.duration;
-          } else if (self.ref.current && !isTimeRelativelySimilar(self.ref.current.currentTime, time, self.ref.current.duration)) {
-            self.ref.current.currentTime = time;
-          }
-        }
-        // Audio v2,v1
-        else {
-          if (self.ref.current) {
-            self.ref.current.currentTime = time;
-          }
-        }
-      },
+    handleSync(data) {
+      if (!self.ref.current) return;
 
-      handleSyncPlay() {
-        // Audio v3
-        if (isFFDev2715) {
-          if (!self.isCurrentlyPlaying) {
-            self.isCurrentlyPlaying = true;
-            try {
-              self.ref.current?.play();
-            } catch {
-              // do nothing, just ignore the DomException
-              // just in case the video was in the midst of syncing
-            }
-          }
-        }
-        // Audio v2,v1
-        else {
-          self.ref.current?.play();
-        }
-      },
+      const video = self.ref.current;
 
-      handleSyncPause() {
-        // Audio v3
-        if (isFFDev2715) {
-          if (self.isCurrentlyPlaying) {
-            self.isCurrentlyPlaying = false;
-            try {
-              self.ref.current?.pause();
-            } catch {
-              // do nothing, just ignore the DomException
-              // just in case the video was in the midst of syncing
-            }
-          }
-        }
-        // Audio v2,v1
-        else {
-          self.ref.current?.pause();
-        }
-      },
+      if (data.playing) {
+        if (!video.playing) video.play();
+      } else {
+        if (video.playing) video.pause();
+      }
 
-      handleSyncDuration(duration) {
-        if (!isFFDev2715) return;
-        if (self.ref.current) {
-          self.setLength(duration * self.framerate);
-        }
-      },
+      if (data.speed) {
+        self.speed = data.speed;
+      }
 
-      handleSyncSpeed(speed) {
-        self.speed = speed;
-      },
+      video.currentTime = data.time;
```

Comment:
It looks like it doesn't affect time at `Timeline`. At least during my investigation sync event from the audio doesn't affect displayed value.

Reply:
it was mysteriously broken because of removed resubscribe to events. most luckily because of weird internal logic dependent on sync operations and local events triggering.